browser/admin/src/AdminSocketSettings.js

```diff
@@ -41,6 +41,21 @@ var AdminSocketSettings = AdminSocketBase.extend({
 					});
 				dialog.open();
 			};
+
+			document.getElementById('btnLogout').onclick = function() {
+				var dialog = (new DlgYesNo())
+					.title(_('Confirmation'))
+					.text(_('Are you sure you want to logout from admin console?'))
+					.yesButtonText(_('OK'))
+					.noButtonText(_('Cancel'))
+					.type('warning')
+					.yesFunction(function() {
+						console.debug(document.cookie);
+						document.cookie = 'JWT= ;expires=Thu, 18 Dec 2013 12:00:00 UTC;max-age=0;path=/;';
```

Comment:
Doesn't work and I have no idea why. Locally in a test file it works: https://jsfiddle.net/ppsilva/o4Lgzf5p/

Reply:
also tried with brand new clean ff profile and still doesn't work.. hm

Reply:
so I think removing this cookie from client side is impossible because it seems to be set with HttpOnly option. so probably only possible to do so in the server side